lib/util/sdp/trackmatcher/identity.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
+ * Construct an {@link IdentityTrackMatcher}.
+ * @class
+ * @classdesc An {@link IdentityTrackMatcher} matches RTCTrackEvents
+ *   with their respective MediaStreamTrack IDs.
+ */
+function IdentityTrackMatcher() {
+  if (!(this instanceof IdentityTrackMatcher)) {
+    return new IdentityTrackMatcher();
+  }
+  Object.defineProperties(this, {
+    _sdp: {
+      value: null,
+      writable: true
+    }
+  });
+}
+
+/**
+ * Match a given MediaStreamTrack with its ID.
+ * @param {RTCTrackEvent} event
+ * @returns {Track.ID}
+ */
+IdentityTrackMatcher.prototype.match = function match(event) {
+  return event.track.id;
+};
+
+/**
+ * Update the {@link IdentityTrackMatcher} with a new SDP.
+ * @param {string} sdp
+ */
+IdentityTrackMatcher.prototype.update = function update(sdp) {
+  this._sdp = sdp;
```

Comment:
We can ignore the SDP argument and skip the `defineProperties` call, IMO. The idea of IdentityTrackMatcher is that it's stateless and that it's just the "identity function" specialized to MediaStreamTrack ID.